site-manage/site-manage-tool/tool/src/bundle/sitesetupgeneric.properties

```diff
@@ -1029,3 +1030,49 @@ gen.listnavselect = To operate the combo box, first press Alt+Down Arrow to open
 search.label=Search
 search.button=Search
 search.clear=Clear Search
+
+help-tips-header=While you wait, enjoy some cheer from the Isidore Cheer Cup!
```

Comment:
These help tips contain an institutional-specific branding of Sakai. The lines should dynamically pull in the local Sakai name from the "ui.service" from sakai.properties.